xrpl/utils/txn_parser/utils/nodes.py

```diff
@@ -16,29 +16,19 @@
 )
 
 
-class OptionalFieldNames(TypedDict, total=False):
-    """The optional fields of `NormalizedNode`."""
-
-    """
-    The fields are separated from `NormalizedNode` to make them optional,
-    while keeping `NodeType`, `LedgerEntryType` and `LedgerIndex` required.
-    """
+class NormalizedNode(TypedDict):
+    """A model representing an affected node in a standard format."""
 
+    NodeType: Literal["CreatedNode", "ModifiedNode", "DeletedNode"]
+    LedgerEntryType: str
+    LedgerIndex: str
     NewFields: Optional[Fields]
     FinalFields: Optional[Fields]
     PreviousFields: Optional[Fields]
     PreviousTxnID: Optional[str]
     PreviousTxnLgrSeq: Optional[int]
```

Comment:
Should these 5 also be `NotRequired` instead of `Optional`?

Reply:
No, those are actually `None`. If you'd rather they not be there instead of having the value of `None`, I can do that.

Reply:
Ah that makes sense - I got a bit confused about the difference between the two :)